packages/twenty-server/src/metadata/relation-metadata/relation-metadata.service.ts

```diff
@@ -200,7 +200,7 @@ export class RelationMetadataService extends TypeOrmQueryService<RelationMetadat
           action: 'alter',
           columns: [
             {
-              action: WorkspaceMigrationColumnActionType.RELATION,
+              action: WorkspaceMigrationColumnActionType.CREATE_RELATION,
```

Comment:
I don't think we should necessary refactor as it was like this before but I feel WorkspaceMigrationColumnActionType.RELATION or CREATE_RELATION is not the right level. It would be clearer if our migrations are closer to postgresSQL API:
- WorkspaceMigrationColumnActionType.CREATE_FOREIGN_KEY


Also for future uses:
- WorkspaceMigrationColumnActionType.CREATE_INDEX
- WorkspaceMigrationColumnActionType.CREATE_CONSTRAINT

regarding index and constraint, as these could be cross columns, we might event want to add a new key next to "columns" in the migration payload and in this case introduce a WorkspaceMigrationIndexActionType and WorkspaceMigrationConstraintActionType

Reply:
@charlesBochet I agree, CREATE_FOREIGN_KEY is actually better since it's literally doing that and only that. Then DROP_FOREIGN_KEY as well. I can change that in this PR